package.json

```diff
@@ -4,7 +4,7 @@
   "private": false,
   "scripts": {
     "build": "npm run bundle-css && npm run build-templates",
-    "build-templates": "./node_modules/.bin/npm-html2js -b src -i 'src/**/*.html' --module templates -o './src/templates.js'",
+    "build-templates": "./node_modules/.bin/npm-html2js -b src -i \"src/**/*.html\" --module templates -o ./src/templates.js\"",
```

Comment:
Single quotes were causing `src/templates.js` to not be generated by the npm build task. Adding escaped double quotes fixed this issue.

Reply:
Hello @bakejeyner , good point - ti sounds like this was causing problems in IntelliJ IDE? I think there may also be a typo, could you try:
"build-templates": "./node_modules/.bin/npm-html2js -b src -i \"src/**/*.html\" --module templates -o \"./src/templates.js\"",
(with the addition of more escaped double quotes)

Reply:
>it sounds like this was causing problems in IntelliJ IDE?

Actually, this was occurring when executing the scripts from the command line:
```
npm run build
```